src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,73 @@
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
+import {Link, useParams} from 'react-router-dom';
+
+import './Subreddit.css';
+
+import thousandsSeparator from '../../helpers/thousandsSeparator';
+
+const Subreddit = ({setSubredditState}) => {
+    const [subredditSpecs, setSubredditSpecs] = useState()
```

Comment:
Omdat `subredditSpecs` uiteindelijk een object zal zijn vind ik hier juist wél dat `undefined` een goeie default waarde is 👍